skopt/tests/test_common.py

```diff
@@ -227,7 +227,7 @@ def test_init_points_and_models(n_random_starts, optimizer_func):
     # starts (no y0 in this case)
     space = [(-5.0, 10.0), (0.0, 15.0)]
     x0 = [[1, 2], [3, 4], [5, 6]]
-    n_calls = 8
+    n_calls = 7
```

Comment:
At the maximum we have `n_random_starts=4` and three initial points so we need seven calls. I think. At least we get an error if I reduce it more :-/